src/test/resources/pages/InstructorEditStudentFeedbackPageModified.html

```diff
@@ -119,9 +119,20 @@
               </span>
             </div>
             <div class="col-sm-12">
-              <textarea class="form-control" cols="100%" name="responsetext-1-0" rows="4">
+              <textarea class="form-control" cols="100%" id="responsetext-1-0" name="responsetext-1-0" onkeydown="updateTextQuestionCharCount('responsetext-1-0', 'responseLength-1-0')" onkeyup="updateTextQuestionCharCount('responsetext-1-0', 'responseLength-1-0')" rows="4">
```

Comment:
Not very high priority, but we should try to move more of the code towards binding event handlers through jQuery instead. So, although most of the code is already using it, let's try to avoid using things like `on.*` in html, if possible.